loadtest/cache.go

```diff
@@ -78,6 +79,14 @@ func (c *cacheImpl) loadInitialData(client lib.KhanInterface) error {
 	return nil
 }
 
+func (c *cacheImpl) getSharedClansPublicIDs() ([]string, error) {
+	var sharedClans []string
+	for _, clan := range c.sharedClans {
+		sharedClans = append(sharedClans, clan.publicID)
```

Comment:
If `c.sharedClans` has 1 million clans, we will use all of than when requesting the summary?

Reply:
Yeah... API will probably respond 4xx in this case. But I don't think this is a huge problem, since you have control on this shared clans input, you can tune it according to your load test.